archinstall/default_profiles/desktops/cinnamon.py

```diff
@@ -12,17 +12,9 @@ def __init__(self) -> None:
 	@override
 	def packages(self) -> list[str]:
 		return [
-			"cinnamon",
-			"system-config-printer",
-			"gnome-keyring",
-			"gnome-terminal",
-			"blueman",
-			"bluez-utils",
-			"engrampa",
-			"gnome-screenshot",
-			"gvfs-smb",
-			"xed",
-			"xdg-user-dirs-gtk"
+			'cinnamon',
+			'gvfs-mtp',
+			'gnome-console'
```

Comment:
What was wrong with all the other packages? It would be great if you could actually provide references and a description for all these changes